askama_derive/src/generator.rs

```diff
@@ -1210,10 +1210,34 @@ impl<'a> Generator<'a> {
         Ok(DisplayWrap::Wrapped)
     }
 
+    #[cfg(not(feature = "serde-json"))]
+    fn _visit_json_filter(
```

Comment:
Style nit: please follow top-down ordering for this -- `_visit_json_filter()` should go below `visit_filter()`. (Same for the others.)

Reply:
Moved around.